workflows/cohort_analysis/cohort_analysis.wdl

```diff
@@ -219,9 +219,10 @@ task merge_and_plot_qc_metrics {
 			echo -e "${sample}\t${adata_path}" >> adata_samples_paths.tsv
 		done < ~{write_lines(preprocessed_adata_objects)}
 
-		python3 /opt/scripts/main/plot_qc_metrics.py \
+		python3 /opt/scripts/main/merge_and_plot_qc.py \
 			--adata-objects-fofn adata_samples_paths.tsv \
 			--adata-output ~{cohort_id}.merged_adata_object.h5ad \
+			--output-metadata-file ~{cohort_id}.initial_metadata.csv \
```

Comment:
@ergonyc this isn't used in the rest of the workflow. Do we just provide this output?

Reply:
Yes, just an output.   pre-filtered metadata.

Reply:
Currently, it's outputted, but it's not a final output for researchers since there's final metadata. Let me know if this needs to be changed

Reply:
@kfang4 we need to add it to the curated output